src/components/Calculator.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-undef */
```

Comment:
- [ ] Kindly check on your tests am getting the error below when I run the test:
![image](https://user-images.githubusercontent.com/61408860/186702198-e9faa78a-9d00-4d85-8ce5-184784014bcb.png)